### PR TITLE
Teacher's guide

### PR DESCRIPTION
**Pull request by bridgethillyer**

### CODE REVIEW

**Comment by bridgethillyer** [comment]
Could people who have taught a workshop recently take a quick look at this and let me know if anything pops out that I should add or change, please? @kpfell @rubygeek

File: TEACHING.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+ClojureBridge Teacher's Guide 
+=============================
+
+There are many ways to teach the ClojureBridge material, and none of them are \"right.\" The following is a guide to working though the curriculum with students based on the experience of some workshop organizers and teachers.
```
Comment (rubygeek) [inline TEACHING.md]: do we need to escape the quotes around right?  \"right.\"

File: TEACHING.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Schedule
+--------
+Friday evening: Installfest - [Getting Set Up](outline/setup.md) in the outline.
+
+Saturday: The rest of the [curriculum](README.md#outline). 
+
+Preparation
+-----------
+Print the [student cheatsheet](outline/cheatsheet.md) and hand out to students at the beginning of the day on Saturday.
```
Comment (rubygeek) [inline TEACHING.md]: add link to pdf version?

**Comment by rubygeek** [comment]
What do ya'll think of putting times by each (or every couple sections) ie 
9a Intro to Clojure
10a Data structures

To make it easier to tell when we should go faster or skip parts to make it to the project?

File: TEACHING.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* [More Simple Values](outline/simple_values2.md)
+* [More Functions](outline/functions2.md)
+* [More Data Structures](outline/data_structures2.md)
+* [Flow Control](outline/flow_control.md)
+
+* [Capstone App Pt. 1](https://github.com/ClojureBridge/drawing/blob/master/curriculum/first-program.md)  -- [Repository](https://github.com/ClojureBridge/drawing/blob/master/README.md)
+* [Capstone App Pt. 2](https://github.com/ClojureBridge/drawing/blob/master/curriculum/create-something.md)
+
+
+Supporting Materials
```
Comment (rubygeek) [inline TEACHING.md]: should we add samples of the info sheet we pass out with links to resources? I liked the one you had for Durham.

File: TEACHING.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+--------------------
+* [Student cheatsheet](outline/cheatsheet.md) | [PDF](ClojureBridgeCheatsheet-v1.pdf)
+* [Slides](http://clojurebridge.github.io/curriculum)
+* [Old global-growth web app](https://github.com/ClojureBridge/global-growth/blob/master/README.md)
+
+Other Curricula
+---------------
+The main [curriculum](#) is well-tested, but there are a number of other curricula available. Feel free to try them out.
```
Comment (davidchambers) [inline TEACHING.md]: Where do you intend the curriculum link to take the reader?

File: TEACHING.md (hunk 1 above)
Comment (kpfell) [inline TEACHING.md]: @rubygeek Ah, no, these don't need to be escaped.

**Comment by kpfell** [comment]
@rubygeek I think it would be really helpful to provide estimated times! Not necessarily as guidelines, but more as, "This is how much time we've found these sections to take with people at X experience level."

**Comment by bridgethillyer** [comment]
Thanks @kpfell, @rubygeek, and @davidchambers for the very helpful feedback! I'd like to merge this soon so London can test it out.

**Comment by kpfell** [comment]
It has a :+1: from me!

**Pull request merged**